Scripts/emails.txt

```diff
@@ -0,0 +1,9 @@
+Joe
```

Comment:
this is dummy_data not a runnable script. Can you move it to scripts/dummy_data/emails.txt. While we're on the subject can you rename the Scripts directory to lowercase. I'm being picky, but I think that's much more normal

Reply:
Done.